www/leaflet-custom.css

```diff
@@ -1,7 +1,12 @@
+html, body { font-size: 100%; }
```

Comment:
What is this doing?

Reply:
Ah, oops! That's part of my template too. I'll remove it.

It makes the `rem` css unit behave  as expected if you need to use it.